### ISSUE TITLE
Await a Python function from Javacript

### ISSUE THREAD

**Issue by RealistikDash** [samuelhwilliams/Eel#152]
Hey there, i am working on a simple discord client using eel. I have a function I call from Javascript that needs to be awaited as it's an async function. So far I haven't found a way to do it. Any help?

**Comment by ChrisKnott** [comment]
Could you explain a bit more about your example? I don't quite follow

**Comment by RealistikDash** [comment]
So I have a function that's something along the lines of
```
@eel.expose
async def sendMessage(message):
    #code
```
For the function to be ran it has to be awaited like `await sendMessage("thing")`. However if in JS I do `eel.sendMessage("thing")` It's ran normally as if I ran it as `sendMessage("thing")` which causes an error.

Sorry in advance if this is also hard to follow.

**Comment by 04fsnape** [comment]
I modified the eelelectron-quick-start folder in demo.

Added this in `hello.html`
```
function run_hello() {
    eel.run_say_hello_py("Javascript World!"); // Call a Python function
 }
run_hello();
```

And `hello.py`

```
import gevent

@eel.expose
def run_say_hello_py(x):
	gevent.spawn(say_hello_py, x)


def say_hello_py(x):
    print('Hello from %s' % x)
```

I hope this helps.
